### DISCUSSION TITLE
How to start Hamburger menu from 768 instead of 640

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#177] [1. Help] [answered]
Hi, Currently Hamburger menu starts with 640px. I want to start it with 768 px.

How can we do that?

**Comment by vovayatsyuk** [comment, marked as answer]
Change the following variables in your theme:

```less
@navigation__media-desktop: ~"(min-width: 768.02px)";
@navigation__media-mobile: ~"(max-width: 768px)";
@navigation__media-wrap: false;
```

You can view other variables at: https://breezefront.com/docs/menu

**Update** 

When using [Swissup_Navigationpro](https://swissuplabs.com/magento-2-navigation-pro-extension.html), add this change as well:

```less
@navpro__sm__max-width: 768px;
```

**Reply by dheerajKsharma** [reply]
Hi @vovayatsyuk ,
That's great. Hamburger comes in and menu also slider from left but menu design is showing of desktop not mobile. Is their any variable for that thing too? In which we can define the design view point of menu as well.

Thanks much.

**Reply by vovayatsyuk** [reply]
These variables affect the default menu only. Do you use some module for the menu? If yes, you need to find similar variables in this module.

**Reply by dheerajKsharma** [reply]
I am using Swissup NavigationPro Module.

**Reply by dheerajKsharma** [reply]
Thanks, I sorted it out. Used variables of Navigation Pro Module. 
Much appreciated!!

**Comment by JamesDenish** [comment]
@dheerajKsharma @vovayatsyuk 

Anyone of you can suggest in which files did you made changes for above solution ?

Thanks in advance.

**Reply by vovayatsyuk** [reply]
You need to create a new less file in your theme: https://breezefront.com/docs/custom-styles
